src/main/java/jpass/ui/JPassFrame.java

```diff
@@ -215,7 +258,7 @@ private JPassFrame(String fileName, Locale locale) {
 
         setJMenuBar(this.jpassMenuBar);
         setDefaultCloseOperation(WindowConstants.DO_NOTHING_ON_CLOSE);
-        setSize(450, 400);
+        setSize(490, 400);
```

Comment:
@gaborbata Increases width to allow long string for some JMenu to be displayed. Let me know what you think about this.

Reply:
Looks good, however it can be a bit wider (plese check with other languages).